protobufs/documents/service.proto

```diff
@@ -31,6 +31,43 @@ service DocumentService {
       description: "Creates a list of precise proofs for the specified fields of the given version of the document given by ID"
     };
   }
+  rpc UpdateDocumentAccessToken(UpdateDocumentAccessTokenRequest) returns (UpdateDocumentAccessTokenResponse) {
```

Comment:
I was thinking on reusing the current invoice/po create/update for this purpose. Is there any major reason to have its own api endpoint?